strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/providers/repository/MavenProxyRepositoryProviderTestIT.java

```diff
@@ -67,65 +67,38 @@
-    {
-        removeRepositories(getRepositories(testInfo));
-    }
-
     @BeforeEach
     public void init(TestInfo testInfo)
             throws Exception
     {
 
         final String repositoryId = getRepositoryName(REPOSITORY_ID, testInfo);
 
-        createProxyRepository(STORAGE_ID,
-                              repositoryId,
-                              REMOTE_URL);
-
         final String centralRepositoryId = getRepositoryName(CENTRAL_REPOSITORY_ID, testInfo);
 
-        createProxyRepository(STORAGE_ID,
-                              centralRepositoryId,
-                              CENTRAL_URL);
-
         artifactEntryService.delete(
                 artifactEntryService.findArtifactList(STORAGE_ID,
-                                                      repositoryId,
-                                                      ImmutableMap.of("groupId", "org.carlspring.maven",
-                                                                      "artifactId", "derby-maven-plugin",
-                                                                      "version", "1.10"),
-                                                      true));
+                        repositoryId,
+                        ImmutableMap.of("groupId", "org.carlspring.maven",
+                                "artifactId", "derby-maven-plugin",
+                                "version", "1.10"),
+                        true));
 
         artifactEntryService.delete(
                 artifactEntryService.findArtifactList(STORAGE_ID,
-                                                      repositoryId,
-                                                      ImmutableMap.of("groupId", "org.carlspring",
-                                                                      "artifactId", "properties-injector",
-                                                                      "version", "1.1"),
-                                                      true));
+                        repositoryId,
+                        ImmutableMap.of("groupId", "org.carlspring",
+                                "artifactId", "properties-injector",
+                                "version", "1.1"),
+                        true));
 
         artifactEntryService.delete(
                 artifactEntryService.findArtifactList(STORAGE_ID,
-                                                      repositoryId,
-                                                      ImmutableMap.of("groupId", "javax.media",
-                                                                      "artifactId", "jai_core",
-                                                                      "version", "1.1.3"),
-                                                      true));
+                        repositoryId,
+                        ImmutableMap.of("groupId", "javax.media",
+                                "artifactId", "jai_core",
+                                "version", "1.1.3"),
```

Comment:
Cosmetic: Please, re-indent all these parameters according to our coding convention.